blues/example.py

```diff
@@ -35,17 +34,17 @@ def runNCMC(platform_name):
 
     #Define the 'model' object we are perturbing here.
     # Calculate particle masses of object to be moved
-    ligand = ncmc.Model(struct, 'LIG')
+    ligand = RandomLigandRotationMove(struct, 'LIG')
     ligand.calculateProperties()
 
     # Initialize object that proposes moves.
-    ligand_mover = ncmc.MoveProposal(ligand, 'random_rotation', opt['nstepsNC'])
+    ligand_mover = MoveEngine(ligand)
```

Comment:
I just commented below on what we had in Slack which I think is more consistent with where we decided to go (https://github.com/MobleyLab/blues/pull/76#issuecomment-308879197). @nathanmlim , your version seems less consistent with what we wanted than Sam's. Am I missing something?